app/controllers/user_registrations_controller.rb

```diff
@@ -31,6 +42,12 @@ def create
     render_error(message: I18n.t('unknown_error', scope: I18N_SCOPE))
   end
 
+  protected
+
+  def check_permissions
+    authorize!(:create, resource)
+  end
```

Comment:
This is actually not needed because everybody is allowed to create an account. This corresponds to:
https://github.com/openfoodfoundation/openfoodnetwork/blob/d5c3edc8fa87f18a4c6adffae87d32618e4363ca/app/models/spree/ability.rb#L37

I think that we should remove this and that line. There might be a similar thing in the v0 API which can be removed.

Reply:
Done @mkllnk I have removed that code.

> There might be a similar thing in the v0 API which can be removed.

I didn't found the similar code in v0 API, Could you please take a look once and confirm if we need any change there.

Reply:
I just had a quick look and we are not creating users in the API. All good. :heavy_check_mark: 

I think that we can now remove this line, too:
https://github.com/openfoodfoundation/openfoodnetwork/blob/d5c3edc8fa87f18a4c6adffae87d32618e4363ca/app/models/spree/ability.rb#L37

Reply:
We have the admin screen to create users but that one is only accessible by super-admin users and they are allowed everything.

Reply:
Hi @mkllnk after removing that permission, few of the specs are failing. Could you please take a look?

Reply:
Ah, thank you for trying that out. It actually highlights some other code which isn't right. I just created another PR to fix one of them. I think for this pull request you should remove that commit again. I'm happy for you to do an interactive rebase and skip that one.

Cleaning up the permissions is our of scope but should be done in another PR some time. But it's too complicated for a beginner, I think.

Reply:
Hi @mkllnk Did you got a chance to look into this?

Reply:
Done @mkllnk Please re-review